imie.txt

```diff
@@ -754,6 +754,25 @@ amap.db.show_binds = true
 
 ------------------------------------------
 
+-- Domyslna opcja startowa czy chodzik jest wlaczony czy nie
+--
+-- Dozwolone wartosci
+-- true/false
+--
+-- Komenda do zmiany w trakcie gry: '/chodzik wlacz' lub '/chodzik wylacz'
+amap.walker_disable = true
```

Comment:
Zmieniłbym tylko na `false` bo ten plik jest używany do stworzenia początkowej konfiguracji - żeby zgrywało się to z defaultem bez pliku.